docs/extensions/logging-and-monitoring.md

```diff
@@ -88,11 +87,12 @@ data:
             description: All infrastructure specific operations cannot be completed (e.g. creating load balancers or persistent volumes).
             summary: Cloud controller manager is down.
 
-  dashboard_operators:
-    <some-json-describing-a-grafana-dashboard-for-operators>
-
-  dashboard_users:
-    <some-json-describing-a-grafana-dashboard-for-users>
```

Comment:
Why these fields are removed now?
Is the new contract backwards-compatible with the existing contract?
I see that extension controllers still use the `dashboard_operators` and `dashboard_users` fields:
- https://github.com/gardener/gardener-extension-provider-aws/blob/f4e3ca6b8b87664c5dbc4e39f5e7fb87360ee3f7/charts/internal/seed-controlplane/charts/csi-driver-controller/templates/configmap-observability.yaml#L9-L15
- https://github.com/gardener/gardener-extension-provider-aws/blob/f4e3ca6b8b87664c5dbc4e39f5e7fb87360ee3f7/charts/internal/seed-controlplane/charts/cloud-controller-manager/templates/configmap-observability.yaml#L9-L61
- https://github.com/gardener/gardener-extension-provider-aws/blob/f4e3ca6b8b87664c5dbc4e39f5e7fb87360ee3f7/charts/internal/machine-controller-manager/seed/templates/configmap-monitoring.yaml#L9-L51

Reply:
Ok, how you guarantee then the backwards-compatibility between `provider-aws < v1.24` and `gardener > v1.24.0`?

Reply:
https://github.com/gardener/gardener/blob/414f94bdf1db7f9cc7b3302f5d395ee0dd83d5d4/charts/seed-monitoring/charts/grafana/templates/grafana-dashboards-configmap.yaml#L36

The old logic still exists and will show the dashboards exposed via the old way. I still removed the documentation about it, but maybe it will be better to keep it until all of the extensions are released and just mark it as deprecated way of exposing dashboards.

Reply:
Ok, thanks for clarifying,